proxy/logging/LogCatUtilsHttp.cc

```diff
@@ -0,0 +1,159 @@
+/** @file
```

Comment:
Does this need its own file? It doesn't fit in any existing file?

Reply:
I originally put it all in LogUtilsHttp.cc, but that pulled a lot of unnecessary stuff into the traffic_logcat link.